### DISCUSSION TITLE
How to install Photoprism

### DISCUSSION THREAD

**Discussion by LateJunction** [photoprism/photoprism#1733] [Q&A] [answered]
I've read every document I can find at docs.photprism.org but somehow can't quite find the page that tells me where to get photoprism, or the page that tells me how to install it, or the page that tells me how to start it, or the page that tells me how to use it.  What am I doing wrong?

**Comment by gstpsk** [comment]
I doubt you've read every document then, It's right there in the side menu under "Setup"! But I'll link it for you [here](https://docs.photoprism.org/getting-started/)

**Comment by graciousgrey** [comment, marked as answer]
The easiest way to set up PhotoPrism is using docker-compose. 
First you need to install docker and docker-compose on your system as described here: https://docs.photoprism.org/getting-started/troubleshooting/#installing-and-using-docker. 
Once docker is installed you need to follow the instructions here: https://docs.photoprism.org/getting-started/docker-compose/. 

> Thanks for the guidance - I failed at the first step: "Download our docker-compose.yml example ..... and change the configuration as needed:" Aside from the file being locked, who is that determines the 'as needed' crtieria ? 

You need to open the .yml file with a text editor, it is important to keep the indentings as shown in the example (https://dl.photoprism.org/docker/docker-compose.yml). 

The comments (text behind a #) in the .yml file tell you what to configure. 

You need to change at least the `PHOTOPRISM_ADMIN_PASSWORD` as well as the path to your files 
```
volumes:
      ## The *originals* folder contains your original photo and video files (- "[host folder]:/photoprism/originals"):
      - "~/Pictures:/photoprism/originals"
```
For examples see: https://docs.photoprism.org/getting-started/docker-compose/#photoprismoriginals

>I've looked through a read only version which might just as well be written in Mandarin. The level of knowledge required to edit this file and still make it valid, while avoiding the obvious security implications, is far beyond me and also far beyond my expectations. It all seems highly impractical for the end-user - a photographer perhaps - to develop this kill and knowledge. Surely there is a better way to start an application under Linux?

We know that it might be a bit tricky for people without much technical background. 
We plan to offer binaries that can be installed with a few clicks and include a nice UI for all configuration steps in the future.
